$NetBSD: patch-aa,v 1.2 2000/09/12 17:23:02 wennmach Exp $

Adopt the location of the config directory to the layout of the
packages system.

--- jconfig.h.orig	Sat Oct  2 14:15:32 1999
+++ jconfig.h	Sat Oct  2 14:17:29 1999
@@ -1,6 +1,6 @@
 /* jconfig.h for Jitterbug. */
 
-#define CONFIG_DIRECTORY "/etc/jitterbug"
+#define CONFIG_DIRECTORY "@PREFIX@/etc/jitterbug"
 
 /* you probably want to leave the rest of these alone */
 #define SYSTEMFILE "/system"
